tests/ui/consts/const-eval/infinite_loop.rs

```diff
@@ -1,8 +1,11 @@
 //! This test tests two things at once:
 //! 1. we error if a const evaluation hits the deny-by-default lint limit
 //! 2. we do not ICE on invalid follow-up code
+//! 3. no ICE when run with `-Z unstable-options` (issue 122177)
 
-//@ compile-flags: -Z tiny-const-eval-limit
+//@revisions: eval_limit no_ice
+//@[no_ice] compile-flags: -Zunstable-options
+//@[eval_limit] compile-flags: -Z tiny-const-eval-limit
```

Comment:
Keep this flag on unconditionally, otherwise the test takes very long to run

Reply:
done